firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/ApkUpdater.java

```diff
@@ -246,8 +246,7 @@ private long downloadToDisk(
     return bytesDownloaded;
   }
 
-  private void validateJarFile(
-      File apkFile, long totalSize, boolean showNotification, long bytesDownloaded)
+  void validateJarFile(File apkFile, long totalSize, boolean showNotification, long bytesDownloaded)
```

Comment:
Please add `@VisibleForTesting`

Reply:
Actually I can add the private back in since we no longer mock this. I don't believe the visibleForTesting annotation is required.

Reply:
good catch!